instrumentation-docs/src/main/java/io/opentelemetry/instrumentation/docs/parsers/SpanParser.java

```diff
@@ -87,7 +87,10 @@ public static Map<String, Map<String, Set<TelemetryAttribute>>> aggregateSpans(
           aggregatedAttributes.computeIfAbsent(when, k -> new HashMap<>());
 
       for (EmittedSpans.SpansByScope spansByScope : spans.getSpansByScope()) {
-        if (spansByScope.getScope().equals(targetScopeName)) {
+        if (spansByScope.getScope().equals(targetScopeName)
+            || scopeAllowList
+                .getOrDefault(targetScopeName, Set.of())
+                .contains(spansByScope.getScope())) {
```

Comment:
I'd suggest not using inheritance for this

instead maybe something like (didn't think too much about names so feel free to pick others)

```suggestion
        if (AllowedScopes.contains(spansByScope, targetScopeName)) {
```